sandbox/SystemDiagnostic.ts

```
@@ -0,0 +1,226 @@
+            variance
+        }
+    }
+
+    private async measurePulseAudioLatency(): Promise<number> {
+        try {
+            // Obtenir info PulseAudio
+            const { stdout } = await execAsync('pactl info')
+            
+            // Chercher "Default Sample Rate" pour estimer latence
+            const sampleRateMatch = stdout.match(/Default Sample Rate: (\d+)/)
+            const sampleRate = sampleRateMatch ? parseInt(sampleRateMatch[1]) : 44100
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)